src/Client/components/KatexMermaid.tsx

```
@@ -2,69 +2,74 @@
-    .join('')
-    .substring(0, 15)
-}
-
+import useTheme from '@/hooks/useTheme'
+
+mermaid.initialize({
+  startOnLoad: false,
+  theme: 'default',
+  securityLevel: 'loose',
+})
+const randomid = () => parseInt(String(Math.random() * 1e15), 10).toString(36)
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)